src/menus.rs

```diff
@@ -110,7 +111,7 @@ fn edit_menu<T: Data>() -> MenuDesc<T> {
                 LocalizedString::new("menu-item-deselect-all").with_placeholder("Deselect All"),
                 consts::cmd::DESELECT_ALL,
             )
-            .hotkey(SysMods::CmdShift, "A"),
+            .hotkey(SysMods::AltCmd, "A"),
```

Comment:
Good catch, this matches system behaviour on mac.

Reply:
I changed it because it conflicts with the shortcut for "align selection", but yes, seems more accurate anyway.